src/pip/_internal/commands/uninstall.py

```diff
@@ -32,7 +35,9 @@ class UninstallCommand(Command, SessionCommandMixin):
       %prog [options] -r <requirements file> ..."""
 
     def __init__(self, *args, **kw):
-        super(UninstallCommand, self).__init__(*args, **kw)
+        # type: (List[Any], Dict[Any, Any]) -> None
+        # https://github.com/python/mypy/issues/4335
+        super(UninstallCommand, self).__init__(*args, **kw)  # type: ignore
```

Comment:
```suggestion
        # type: (Any, Any) -> None
        super(UninstallCommand, self).__init__(*args, **kw)
```

Does this work?

Reply:
As mentioned in https://github.com/pypa/pip/pull/8018#issuecomment-612725333, I think the line above should be `*Any, **Any`, then it'll work.

Reply:
So both `Any, Any` and `*Any, **Any` work, but I am not sure which one is more appropriate. 

According to https://github.com/python/mypy/issues/5876 , the example provided is `def f(self, *args: Any, **kwargs: Any) -> int: pass` , which I think is closer to 
`(Any, Any) -> None` in our case of `def __init__(self, *args, **kw):` rather than `(*Any, **Any`) -> None`

Reply:
I have changed it to `(*Any, **Any) -> None`